frontend/src/components/pages/connect/helper.tsx

```diff
@@ -704,21 +705,29 @@ export const TaskState = observer((p: { observable: { state: ClusterConnectorTas
         </Button>
 
         const close = () => showErr(undefined);
-        errModal = <Modal open={err != null} onOk={close} onCancel={close} cancelButtonProps={{ style: { display: 'none' } }}
-            bodyStyle={{ paddingBottom: '8px', paddingTop: '14px' }}
-            centered
-            closable={false} maskClosable={true}
-            okText="Close" width="60%"
-        >
-            <>
-                {
-                    task.taskId == null
-                        ? <h3>Error in Connector</h3>
-                        : <h3>{`Error trace of task ${task.taskId}`}</h3>
-                }
-                <div className="codeBox" style={{ whiteSpace: 'pre', overflow: 'scroll', width: '100%', padding: '10px 8px' }}>{err}</div>
-            </>
-        </Modal>;
+        errModal = (
+            <Modal
+                isOpen={err != null}
+                onClose={close}
+            >
+                <ModalOverlay />
+                <ModalContent minW="5xl">
+                    <ModalHeader>
+                        {
+                            task.taskId == null
+                                ? <>Error in Connector</>
+                                : <>{`Error trace of task ${task.taskId}`}</>
```

Comment:
I see we replaced `<h3>` by `<>`, maybe we can do `<Text as="h3">` or use the new variant? Or just revert back to `h3` and do the change later.

Reply:
I think this is handled by `<ModalHeader>`, but I transformed these fragments into strings